packages/components/nodes/documentloaders/Cheerio/Cheerio.ts

```diff
@@ -90,6 +90,7 @@ class Cheerio_DocumentLoaders implements INode {
         const textSplitter = nodeData.inputs?.textSplitter as TextSplitter
         const metadata = nodeData.inputs?.metadata
         const relativeLinksMethod = nodeData.inputs?.relativeLinksMethod as string
+        const selectedLinks = nodeData.inputs?.selectedLinks as string[]
         let limit = nodeData.inputs?.limit as string
```

Comment:
can we set the default value in the inputs to 10 ?

Reply:
Set the default value for the limit field to `10`.